geometry_msgs/PoseStamped pose
string object_id
